sys/stat.h used to be indirectly included through
device/udev_linux/scoped_udev.h -> libudev.h -> sys/stat.h
but libudev.h in jethro no longer includes sys/stat.h resulting in the
following compile time errors:

arm-angstrom-linux-gnueabi-g++ ...  ../../components/storage_monitor/storage_monitor_linux.cc -o obj/components/storage_monitor/storage_monitor.storage_monitor_linux.o
../../components/storage_monitor/storage_monitor_linux.cc: In function 'scoped_ptr<storage_monitor::StorageInfo> storage_monitor::{anonymous}::GetDeviceInfo(const base::FilePath&, const base::FilePath&)':
../../components/storage_monitor/storage_monitor_linux.cc:132:15: error: aggregate 'storage_monitor::{anonymous}::GetDeviceInfo(const base::FilePath&, const base::FilePath&)::stat device_stat' has incomplete type and cannot be defined
   struct stat device_stat;
               ^
../../components/storage_monitor/storage_monitor_linux.cc:133:53: error: invalid use of incomplete type 'struct storage_monitor::{anonymous}::GetDeviceInfo(const base::FilePath&, const base::FilePath&)::stat'
   if (stat(device_path.value().c_str(), &device_stat) < 0)
                                                     ^
../../components/storage_monitor/storage_monitor_linux.cc:132:10: error: forward declaration of 'struct storage_monitor::{anonymous}::GetDeviceInfo(const base::FilePath&, const base::FilePath&)::stat'
   struct stat device_stat;
          ^
../../components/storage_monitor/storage_monitor_linux.cc:137:34: error: 'S_ISCHR' was not declared in this scope
   if (S_ISCHR(device_stat.st_mode))
                                  ^
../../components/storage_monitor/storage_monitor_linux.cc:139:39: error: 'S_ISBLK' was not declared in this scope
   else if (S_ISBLK(device_stat.st_mode))
                                       ^

As the code makes use of stat systemcalls include the file explicitely.
Tested with ARCH armv7.

Signed-off-by: Max Krummenacher <max.oss.09 at gmail.com>
Upstream-Status: Pending

--- chromium-53.0.2774.4-ORIG/components/storage_monitor/storage_monitor_linux.cc       2016-06-22 12:04:10.000000000 -0400
+++ chromium-53.0.2774.4/components/storage_monitor/storage_monitor_linux.cc    2016-06-24 17:43:39.874640219 -0400
@@ -9,6 +9,7 @@
 #include <mntent.h>
 #include <stdint.h>
 #include <stdio.h>
+#include <sys/stat.h>
 #include <limits>
 #include <list>
 #include <utility>
